### PR TITLE
8.0 document page quality manual

### PR DESCRIPTION
**Pull request by max3903**
Depends on:
- [x] #23

### CODE REVIEW

File: document_page_quality_manual/__openerp__.py [hunk 1]
```diff
@@ -25,20 +25,25 @@
     'category': 'Management System',
     'complexity': "easy",
     'description': """
```
Comment (pedrobaeza) [inline document_page_quality_manual/__openerp__.py:27]: This is not needed as you have put the README.rst file.

**Comment by pedrobaeza** [comment]
Please remove migration scripts and change icon path.

**Comment by bwrsandman** [comment]
Needs a rebase

**Comment by llacroix** [comment]
@max3903 could you add those dependencies in the description

Depends on:
- [x] #23

**Comment by dreispt** [comment]
:+1:

**Comment by max3903** [comment]
:+1: 

Tested on runbot.

**Pull request merged**